OutOfSchool/OutOfSchool.WebApi/Controllers/PhotoController.cs

```diff
@@ -0,0 +1,478 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.StaticFiles;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Localization;
+using OutOfSchool.Services.Enums;
+using OutOfSchool.WebApi.Extensions;
+using OutOfSchool.WebApi.Models;
+using OutOfSchool.WebApi.Services.PhotoStorage;
+
+namespace OutOfSchool.WebApi.Controllers
+{
+    [ApiController]
+    [Route("[controller]/[action]")]
+    [Authorize(AuthenticationSchemes = "Bearer")]
+    public class PhotoController : ControllerBase
+    {
+        private readonly IPhotoStorage photoService;
+        private readonly IContentTypeProvider contentTypeProvider;
+        private readonly IStringLocalizer<SharedResource> localizer;
+        private readonly long maxSize;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="PhotoController"/> class.
+        /// </summary>
+        /// <param name="photoService">Service for Photo model.</param>
+        /// <param name="contentTypeProvider">Provides a mapping between file extensions and MIME types.</param>
+        /// <param name="localizer">Localizer.</param>
+        /// <param name="config">Config.</param>
+        public PhotoController(IPhotoStorage photoService, IContentTypeProvider contentTypeProvider, IStringLocalizer<SharedResource> localizer, IConfiguration config)
+        {
+            this.localizer = localizer;
+            this.photoService = photoService;
+            this.contentTypeProvider = contentTypeProvider;
+            this.maxSize = config.GetValue<long>("PhotoSettings:MaxSizeBytes");
+        }
+
+        /// <summary>
+        /// Get file by it's name.
+        /// </summary>
+        /// <param name="fileName">Name of the file.</param>
+        /// <returns>Photo.</returns>
+        [HttpGet]
+        [Authorize(Roles = "parent")]
+        [Authorize(Roles = "provider")]
+        [Authorize(Roles = "admin")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
```

Comment:
For this and other endpoints if using `IActionResult` - please specify response types in attributes so swagger documents generate correct data.